tests/Api/CreateProductTest.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+namespace Akeneo\Pim\ApiClient\tests\Api;
+
+use Akeneo\Pim\ApiClient\Api\ProductApi;
+use donatj\MockWebServer\Response;
+use donatj\MockWebServer\ResponseStack;
+
+class CreateProductTest extends ApiTestCase
+{
+    public function test_create_product()
+    {
+        $this->server->setResponseOfPath(
+            '/'. sprintf(ProductApi::PRODUCTS_URI, 'black_sneakers'),
+            new ResponseStack(
+                new Response('', [], 201)
+            )
+        );
+
+        $api = $this->createClient()->getProductApi();
+        $response = $api->create('new_shoes', $this->newProduct());
+
+        $this->assertSame(201, $response);
```

Comment:
same (almost evrywhere): check the body of the request. You can do a function in `ApiTestCase`.